src/site/content/en/blog/user-agent-client-hints/index.md

```diff
@@ -0,0 +1,434 @@
+| `UA-Platform`              | `Sec-CH-UA-Platform`         | `"Android"`                                      | The platform for the device, usually the operating system (OS).                                                                                                              |
+| `UA-Platform-Version`      | `Sec-CH-UA-Platform-Version` | `"10"`                                           | The version for the platform or OS.                                                                                                                                          |
+| `UA-Arch`                  | `Sec-CH-UA-Arch`             | `"ARM64"`                                        | The underlying architecture for the device. While this may not be relevant to displaying the page, the site may want to offer a download which defaults to the right format. |
+| `UA-Model`                 | `Sec-CH-UA-Model`            | `"Pixel 3"`                                      | The device model.                                                                                                                                                            |
+
+{% Aside 'gotchas' %}
+  Privacy and compatibility considerations mean the value may be blank, not
+  returned, or populated with a varying value. This is referred to as
+  [GREASE](https://wicg.github.io/ua-client-hints/#grease).
+{% endAside %}
+
+### Example exchange
+
+An example exchange would look like this:
+
+⬆️ _Initial request from browser_<br> The browser is requesting the `/downloads`
+page from the site and sends its default basic User-Agent.
+
+```text
+GET /downloads HTTP/1.1
+Host: example.site
+
+Sec-CH-UA: "Chromium";v="84", "Google Chrome";v="84"
+Sec-CH-UA-Mobile: ?0
+```
+
+⬇️ _Response from server_<br> The server sends the page back and additionally
+asks for the full browser version and the platform.
+
+```text
+HTTP/1.1 200 OK
+Accept-CH: UA-Full-Version, UA-Platform
+
+<html>…
+```
+
+⬆️ _Subsequent requests_<br> The browser grants the server access to the
+additional information and sends the extra hints back in all subsequent
+responses.
+
+```text
+GET /downloads/app1 HTTP/1.1
+Host: example.site
+
+Sec-CH-UA: "Chromium";v="84", "Google Chrome";v="84"
+Sec-CH-UA-Mobile: ?0
+Sec-CH-UA-Full-Version: "84.0.4143.2"
+Sec-CH-UA-Platform: "Android"
+```
+
+### JavaScript API
+
+Alongside the headers, the User-Agent can also be accessed in JavaScript via
+`navigator.userAgentData`. The default `Sec-CH-UA` and `Sec-CH-UA-Mobile` header
+information can be accessed via the `brands` and `mobile` properties,
+respectively:
+
+```js
+console.log(navigator.userAgentData.brands);
+
+[
+  {
+    brand: 'Chromium',
+    version: '84',
+  },
+  {
+    brand: 'Google Chrome',
+    version: '84',
+  },
+];
+
+console.log(navigator.userAgentData.mobile);
+
+false;
```

Comment:
```suggestion
console.log(navigator.userAgentData.brands);

// [
//   {
//     brand: 'Chromium',
//     version: '84',
//   },
//   {
//     brand: 'Google Chrome',
//     version: '84',
//   },
// ];

console.log(navigator.userAgentData.mobile);

// false;
```
I'm not married to this exact convention but I think we need to provide some indicator that this is output. Because on my first readthrough when I saw `false` I thought you had accidentally copied some unrelated code.

Reply:
I'm trying a code comment before the input and output lines to make a clearer separation and still get the syntax highlighting on the output.